mars/dataframe/expressions/datasource/from_tensor.py

```diff
@@ -0,0 +1,94 @@
+from ..core import DataFrameOperand, DataFrameOperandMixin, ObjectType
+from ..utils import parse_index
+from .... import opcodes as OperandDef
+from ....serialize import KeyField, SeriesField
+import numpy as np
+
+try:
+    import pandas as pd
+except ImportError:  # pragma: no cover
+    pass
+
+
+class DataFrameFromTensor(DataFrameOperand, DataFrameOperandMixin):
+    """
+    Represents data from mars tensor
+    """
+    _op_type_ = OperandDef.DATAFRAME_FROM_TENSOR
+    _dtypes = SeriesField('dtypes')
+    _input = KeyField('input')
+
+    def __init__(self, dtypes=None, gpu=None, sparse=None, **kw):
+        super(DataFrameFromTensor, self).__init__(_dtypes=dtypes,
+                                                  _gpu=gpu, _sparse=sparse,
+                                                  _object_type=ObjectType.dataframe, **kw)
+
+    @property
+    def dtypes(self):
+        return self._dtypes
+
+    @property
+    def input(self):
+        return self._input
+
+    def _set_inputs(self, inputs):
+        super(DataFrameFromTensor, self)._set_inputs(inputs)
+        self._input = inputs
+
+    def __call__(self, input_tensor):
+        if input_tensor.ndim == 0:
+            raise IndexError('Not support converted from 0-dim tensor')
+        elif input_tensor.ndim == 1:
+            # convert to Series
```

Comment:
It is also a DataFrame type when `ndim` is 1

Reply:
hi, Kaisheng. I have the wrong comments. I mean that convert to 1-d dataframe (only 1 column). Is it OK? (Not convert to series in this function)

Reply:
Yeah, I think it's just doc issue.